cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/TextAnalytics.scala

```diff
@@ -609,4 +633,10 @@ class TextAnalyze(override val uid: String) extends TextAnalyticsBase(uid)
     )
   }
 
+  // total kluge for R
+  private def convertJsonToTasks(jsonString: String): Seq[TextAnalyzeTask] = {
+    implicit val formats = DefaultFormats
+    val p = parse(jsonString).extract[Seq[Map[String, Map[String,String]]]]
+    Seq[Map[String, String]](p.head.head._2).map(m => TextAnalyzeTask(m))
+  }
```

Comment:
This one was particularly awful. I couldn't find a lower level way to pass the argument.

Reply:
FYI, what I did in dotnet is creating a corresponding `TextAnalyzeTask` object from dotnet side, that maps into the `TextAnalyzeTask` in scala. And the constructor I used is https://github.com/microsoft/SynapseML/blob/master/cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/TextAnalyticsAnalyzeSchemas.scala#:~:text=def%20this(,%7D  ; if R can map to java.util.hashMap, you can invoke this method to construct an object and get the underlying java object, then pass it in from R side.